sunpy/map/mapsequence.py

```diff
@@ -81,6 +81,16 @@ def __len__(self):
         """Return the number of maps in a mapsequence."""
         return len(self.maps)
 
+    def __repr__(self):
+        names = []
+        for m in self.maps:
+            name = m.nickname + 'Map'
+            if name not in names:
+                names.append(name)
+
+        no_of_elements = len(self.maps)
+        return 'MapSequence of {} elements '.format(no_of_elements) + str(tuple(names))
```

Comment:
```suggestion
        return f'MapSequence of {no_of_elements} with maps from {", ".join(names)}'
```
Switch to an f-string and have a cleaner look for the name list.